bode/bode/models/tag.py

```diff
@@ -38,5 +31,11 @@ def delete(tag_id):
 
         return tag
 
+    def getByName(tag_name):
```

Comment:
That name should probably be snake_cased

Reply:
Yeap, sorry. It's a habit